pythonfuzz/dictionary.py

```diff
@@ -46,16 +46,18 @@ def load_file(self, dict_path):
 
         _dict = set()
         with open(dict_path) as f:
-            for line in f:
+            for nb, line in enumerate(f):
                 line = line.lstrip()
                 if not line or line.startswith('#'):
                     continue
                 word = self.line_re.search(line)
-                if word:
-                    # Decode any escaped characters, giving us a bytes object
-                    value = word.group(1)
-                    (value, _) = codecs.escape_decode(value)
-                    _dict.add(value)
+                if not word:
+                    print('Error while loading the dictionary "%s" at line %d.'
+                            % (dict_path, line))
```

Comment:
Probably want to include a `continue` here, as otherwise it'll exception on the next line when it tries to get the 'group' method of None.